src/benchmarks/micro/corefx/System.Text.Json/Serializer/ReadJson.GenericCollection.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class WriteGenericCollection
```

Comment:
File mismatch. `ReadJson.X` has the test for Write and vice versa.

Reply:
Is there another commit coming that resolves this @layomia ?